File: src/main/java/com/dh/ondot/member/core/JwtProperties.java
package com.dh.ondot.member.core;

import org.springframework.boot.context.properties.ConfigurationProperties;

@ConfigurationProperties(prefix = "jwt")
public record JwtProperties(
        String secret,
        long accessTokenExpireTimeInHours,
        long refreshTokenExpireTimeInHours
) {
}

File: src/main/kotlin/com/dh/ondot/member/core/JwtProperties.kt
package com.dh.ondot.member.core

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties(prefix = "jwt")
data class JwtProperties(
    val secret: String,
    val accessTokenExpireTimeInHours: Long,
    val refreshTokenExpireTimeInHours: Long,
)
